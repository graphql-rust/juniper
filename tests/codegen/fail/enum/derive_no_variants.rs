#[derive(juniper::GraphQLEnum)]
enum Test {}

fn main() {}
