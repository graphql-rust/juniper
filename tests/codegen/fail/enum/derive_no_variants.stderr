error: GraphQL enum expected at least 1 non-ignored variant
 --> fail/enum/derive_no_variants.rs:1:10
  |
1 | #[derive(juniper::GraphQLEnum)]
  |          ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `juniper::GraphQLEnum` (in Nightly builds, run with -Z macro-backtrace for more info)
